xml/deployment_cifs.xml

```diff
@@ -0,0 +1,290 @@
+     Create a directory for the CTDB lock on the shared file system:
+    </para>
+<screen>&prompt.root;<command>mkdir</command> -p /srv/samba/</screen>
+   </step>
+   <step>
+    <para>
+     In <filename>/etc/ctdb/nodes</filename> insert all nodes which contain
+     all private IP addresses of each node in the cluster:
+    </para>
+<screen>&subnetI;.1
+&subnetI;.2</screen>
+   </step>
+   <step>
+    <para>
+     Configure Samba. Add the following lines in the
+     <literal>[global]</literal> section of
+     <filename>/etc/samba/smb.conf</filename> on &si.earth; and &si.mars;.
+     Use the host name of your choice in place of "CTDB-SERVER" (all nodes
+     in the cluster will appear as one big node with this name, effectively):
+    </para>
+<screen>[global]
+    # ...
+    # settings applicable for all CTDB deployments
+    netbios name = CTDB-SERVER
+    clustering = yes
+    idmap config * : backend = tdb2
+    passdb backend = tdbsam
+    ctdbd socket = /var/lib/ctdb/ctdb.socket
+    # settings necessary for CTDB on OCFS2
+    fileid:algorithm = fsid
+    vfs objects = fileid
+    # ...</screen>
+    <para>
+     Alternatively, <command>csync2</command> can be used to synchronize
+     config files. For details, see
+     <link xlink:href=
+     "https://www.suse.com/documentation/sle-ha-12/singlehtml/book_sleha/book_sleha.html#pro.ha.installation.setup.csync2.start"
+     />.
+    </para>
+   </step>
+   <step>
+    <para>
+     Register the &sle; &hasi; on &si.earth; and &si.mars;.
+    </para>
+    <screen>
+&prompt.root;<command>SUSEConnect</command> -r <replaceable>ACTIVATION_CODE</replaceable> -e <replaceable>E_MAIL</replaceable>
+    </screen>
+   </step>
+   <step>
+    <para>
+     Install <package>ha-cluster-bootstrap</package> on both nodes:
+    </para>
+    <screen>
+&prompt.root;<command>zypper</command> in ha-cluster-bootstrap
+    </screen>
+   </step>
+   <step>
+    <substeps>
+     <step>
+      <para>
+       Init the cluster on &si.earth;:
+      </para>
+      <screen>
+&prompt.earth;<command>ha-cluster-bootstrap</command>
+      </screen>
+     </step>
+     <step>
+      <para>
+       Let &si.mars; join the cluster:
+      </para>
+      <screen>
+&prompt.mars;<command>ha-cluster-join</command> -c earth
+      </screen>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     Check the status of cluster. You should see two nodes added in cluster:
+    </para>
+    <screen>
+&prompt.earth;<command>crm</command> status
+    </screen>
+   </step>
+   <step>
+    <para>
+     Execute the following commands on &si.earth; to configure the the CTDB resource:
+    </para>
+<!--taroth 2012-02-07: fixed bnc#745334, default timeout for start/stop-->
+<screen>&prompt.earth;<command>crm</command> configure
+&prompt.crm.conf;<command>primitive</command> ctdb ocf:heartbeat:CTDB params \
+    ctdb_manages_winbind="false" \ 
+    ctdb_manages_samba="false" \
+    ctdb_recovery_lock="/srv/samba/ctdb.lock" \
```

Comment:
Maybe something for developers to answer: I'm not sure, but should the lock somewhere in `/var/lock/....`?